src/Cli/dotnet/commands/dotnet-tool/install/LocalizableStrings.resx

```diff
@@ -170,6 +170,9 @@ Tool '{1}' (version '{2}') was successfully installed. Entry is added to the man
   <data name="LocalOptionDescription" xml:space="preserve">
     <value>Install the tool and add to the local tool manifest (default).</value>
   </data>
+  <data name="WorkloadSetMode" xml:space="preserve">
+    <value>Determine whether dotnet should use workload sets or loose manifests when installing or updating workloads.</value>
+  </data>
```

Comment:
I think this string is in the wrong .resx file (this one is for tool installation, not workload installation).  It looks the string is also in the right .resx, so I think this can just be deleted.